README.md

```diff
@@ -102,39 +102,9 @@ JEKYLL_ENV=production bundle exec rake generate
 
 ## Updating Code Documentation
 
-To update the documentation, you can run the script `./dev/gen_apidocs.sh` in
-the `apache/arrow` repository. This script will run the code documentation
-tools in a fixed environment.
-
-### C (GLib)
-
-First, build Apache Arrow C++ and Apache Arrow GLib. This assumes that you have checkouts your forks of `arrow` and `arrow-site` alongside each other in your file system.
-
-```
-mkdir -p ../cpp/build
-cd ../cpp/build
-cmake .. -DCMAKE_BUILD_TYPE=debug
-make
-cd ../../c_glib
-./autogen.sh
-./configure \
-  --with-arrow-cpp-build-dir=$PWD/../cpp/build \
-  --with-arrow-cpp-build-type=debug \
-  --enable-gtk-doc
-LD_LIBRARY_PATH=$PWD/../cpp/build/debug make GTK_DOC_V_XREF=": "
-rsync -r doc/reference/html/ ../../arrow-site/asf-site/docs/c_glib/
-```
-
-### JavaScript
-
-```
-cd ../js
-npm run doc
-rsync -r doc/ ../../arrow-site/asf-site/docs/js
-```
-
-Then add/commit/push from the `asf-site/` git checkout.
-
+This is built in the repositories corresponding to each code and then copied into
+the website repository before deploying the website.  Separate tooling is used for
+creating the code documentation.
```

Comment:
I think that we can remove this section.
Some repositories such as `apache/arrow` and `apache/arrow-ballista` copy their documents to this repository but others such as `apache/arrow-adbc` and `apache/arrow-datafusion` uses their repositories. See also: https://github.com/apache/arrow-adbc/blob/1e1a57623761d564ac4a99a213fbeb4142b4b57a/.asf.yaml#L39-L41

I don't think that we need to describe them here because each repository should have documentation for it.

Reply:
Removed the section.